test/sql-tap/lua/sqltester.lua

```diff
@@ -198,6 +198,19 @@ local function do_catchsql_test(self, label, sql, expect)
 end
 test.do_catchsql_test = do_catchsql_test
 
+local function do_catchsql_prefix_test(self, label, sql, prefix_end, expect)
```

Comment:
Do you really need this function for single test? I'd better fix that test than introducing new sql-tap function

Reply:
This test runs 2 times(under vinyl and under memtx) and one of tuple fields depends on the run number. One of the ways to fix it - don't show these tuples. I found only one way to do it - add function.